src/components/AcceptAll.tsx

```diff
@@ -31,17 +31,20 @@ export default function AcceptAll({
   const { formatMessage } = useIntl();
 
   // Opt in to all purposes
-  const handleAcceptAll: JSX.MouseEventHandler<HTMLButtonElement> | undefined =
-    (event: JSX.TargetedEvent<HTMLButtonElement, MouseEvent>): void => {
-      event.preventDefault();
-      const purposeTypes = airgap.getPurposeTypes();
-      const consent: TrackingConsent = {};
-      Object.keys(purposeTypes).forEach((purpose) => {
-        consent[purpose] = true;
-      });
-      airgap.setConsent(event, consent);
-      handleSetViewState('close');
-    };
+  const handleAcceptAll:
+    | JSX.MouseEventHandler<HTMLButtonElement>
+    | undefined = (
```

Comment:
is this confirmed ok w/o event click @eligrey ?

Reply:
this didnt change its just prettier, see other comment

Reply:
if event was undefined it would fail higher up with event.preventDefault() anyways so I think this is fine